### PR TITLE
feat: implement browsingContext.activate

### PR DESCRIPTION
**Pull request by OrKoN**
WPT: https://github.com/web-platform-tests/wpt/pull/40964
Spec: https://github.com/w3c/webdriver-bidi/pull/453

### CODE REVIEW

File: src/bidiMapper/domains/context/browsingContextProcessor.ts [hunk 1]
```diff
@@ -300,6 +300,19 @@ export class BrowsingContextProcessor {
     );
   }
 
+  async process_browsingContext_activate(
+    params: BrowsingContext.ActivateParameters
+  ): Promise<EmptyResult> {
+    const context = this.#browsingContextStorage.getContext(params.context);
+    if (!context.isTopLevelContext()) {
+      throw new InvalidArgumentException(
+        'Emulating viewport is only supported on the top-level context'
```
Comment (thiagowfx) [inline src/bidiMapper/domains/context/browsingContextProcessor.ts:309]: nit: I know it's still a draft, but just FYI, this error message is too specific. It should not mention viewport (e.g. we could be calling `activate` for screenshot purposes).

Comment (OrKoN) [reply src/bidiMapper/domains/context/browsingContextProcessor.ts:309]: yeah, I just copy pasted it from the other command, still in progress.

**Comment by thiagowfx** [comment]
TODO for you (either in this PR or in the next one, but keep it in mind): Update

https://github.com/GoogleChromeLabs/chromium-bidi/blob/cfa01d1b2ec867e2b47d49f20f932f477631a411/src/bidiMapper/domains/context/browsingContextImpl.ts#L719

...to use the bidi API instead of calling the CDP command directly.

**Comment by thiagowfx** [comment]
LGTM (adding one test would be nice).

**Review by jrandolf — APPROVED** [review]
+1 to adding a test

**Comment by OrKoN** [comment]
Added a basic test (more extensive test coverage will be done by WPT)

File: tests/test_activate.py
Comment (thiagowfx) [inline tests/test_activate.py:1]: I would have suggested to add this to the browsing context test file.

The top-level directory for tests/* becomes a bit messy if we put everything here.

Alternatively, we could mirror WPT and create a top-level browsing_context dir, and move this therein.

(Feel free to re-request review before submitting, I tend to review fast and would have suggested this in advance.)

Comment (OrKoN) [reply tests/test_activate.py:1]: That sounds like a separate issue. Putting everything into the single browsing context test file is messy too.

Comment (thiagowfx) [reply tests/test_activate.py:1]: Do you think it's worth to create many subdirectories, like WPT? We could start to slowly migrate our codebase to a similar pattern.

Comment (OrKoN) [reply tests/test_activate.py:1]: I think subdirectories is the way to go where the directory name is the module name. I think we should not create a directory per command yet (a file per command/event should be okay)

**Pull request merged**